lib/times.js

```diff
@@ -40,7 +40,9 @@ function create (types) {
 }
 
 var times = {
-  week: function ( ) { return create('weeks')(1); }
+	mins_msecs: function (value) { return value*60*1000; }
```

Comment:
I have it done in my daw_001 branch but I am unsure how to get it into this one.